### DISCUSSION TITLE
What is the data structure?

### DISCUSSION THREAD

**Discussion by kpagcha** [Drarig29/brackets-viewer.js#100] [Q&A]
What data structure is required for the viewer? Let's say I have my backend with matches, scores, team names, etc. How do I prepare the data to pass to the viewer directly and render my brackets? For the love of my I can't find this in the documentation.

**Comment by Drarig29** [comment]
You have to use the https://github.com/Drarig29/brackets-manager.js/ library.

You should find it in the documentation: https://drarig29.github.io/brackets-docs/getting-started/
